### PR TITLE
Create the withdraw info popup.

### LINKED ISSUE #92: Create the withdraw info popup
As part of the `/withdraw` flow, we need to create a basic popup to collect the required withdrawal information and update the transaction in the backend. The pointers in #34 largely apply to this task as well.

### PR DESCRIPTION
**Pull request by debnil**
This PR creates the popup to collect information about a withdrawal (the amount, bank account, and name of the bank). It closes #92.

### CODE REVIEW

File: src/templates/withdraw/success.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+{% block "title" %}
+<title>Successful Withdrawal!</title>
+{% endblock %}
+
+
+{% block "content" %}
+<section class="section">
+    <p>
+        Success! You may now close this window. A postMessage has also been
+        published to either window.opener or window.parent.
```
Comment (msfeldstein) [inline src/templates/withdraw/success.html]: We don't need to mention anything about postMessage, we should just show a withdraw receipt.

File: src/templates/withdraw/success.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    </p>
+
+    <script type="text/javascript">
+        function postTransactionCallback() {
```
Comment (msfeldstein) [inline src/templates/withdraw/success.html]: Where is this called?

File: src/withdraw/views.py [hunk 3]
```diff
@@ -26,10 +33,60 @@ def _construct_interactive_url(request, transaction_id):
...
+@xframe_options_exempt
+@api_view(["GET", "POST"])
 def interactive_withdraw(request):
-    # TODO: Fill in interactive withdrawal view.
-    return Response()
+    """
+    `GET /withdraw/interactive_withdraw` opens a form used to input information about
+    the withdrawal. This creates a corresponding transaction in our database.
+    """
+    transaction_id = request.GET.get("transaction_id")
```
[the remark this answers is not in the document]
Comment (debnil) [reply src/withdraw/views.py]: Yup, it can still be used to access query parameters.

File: src/templates/withdraw/success.html (hunk 2 above)
Comment (debnil) [reply src/templates/withdraw/success.html]: Good catch, should be called below now.

File: src/templates/withdraw/success.html (hunk 1 above)
Comment (debnil) [reply src/templates/withdraw/success.html]: Do we want to navigate to a receipt here? I thought the receipt would be the `more-info-url`, and instead this should display what's happening in the flow.

File: src/withdraw/views.py [hunk 4]
```diff
@@ -26,10 +33,64 @@ def _construct_interactive_url(request, transaction_id):
...
+        asset = Asset.objects.get(name=asset_code)
+        form.asset = asset
+
+        # If the form is valid, we create a transaction pending user action
+        # and render the success page.
+        if form.is_valid():
+            amount_in = form.cleaned_data["amount"]
+            amount_fee = calc_fee(asset, settings.OPERATION_WITHDRAWAL, amount_in)
+            # We copy the UUID to satisfy length requirements of the memo.
+            withdraw_memo = uuid.UUID(transaction_id).hex * 2
```
Comment (msfeldstein) [inline src/withdraw/views.py]: can you zero pad this instead?  It'd make it a bit more obvious whats happening when someone looks at the memo.

**Pull request merged**